app/models/manageiq/providers/google/cloud_manager.rb

```diff
@@ -16,7 +16,6 @@ class ManageIQ::Providers::Google::CloudManager < ManageIQ::Providers::CloudMana
   include ManageIQ::Providers::Google::ManagerMixin
 
   supports :provisioning
-  supports :regions
 
   before_create :ensure_managers
   before_update :ensure_managers_zone_and_provider_region
```

Comment:
@bronaghs Why should we keep it there? GCE has currently no usecase for regions at all.

Reply:
Im referring to line 21, Im implying it looks like it isnt.

Reply:
Ah, I see, sorry.

Well, this will keep the manager zones in sync and I assume we want that. Also as a sideeffect it would sync the provider_region, sure. Though if it's already `nil`, it's `nil` for both managers and nothing really happens.

The other way would be to define a new `before_update` rule, which would sync only the zone. And also new `before_create` rule, since `:ensure_managers` uses `:ensure_managers_zone_and_provider_region` internally. I just thing leaving it there is harmless. What do you think?